src/bin/opensmt.cc

```diff
@@ -238,4 +243,42 @@ void catcher( int sig )
   }
 }
 
+
+void printHelp( )
+{
+    const char help_string[]
+        = "Usage: ./opensmt [OPTION] filename\n"
+          "where OPTION can be\n"
+          "  --help [-h]              print this help\n"
+          "  --config=<filename>      use configuration file <filename>\n";
+    std::cerr << help_string;
+    exit( 1 );
+}
+
+void printVersion( )
+{
+    char version_string[]
+        = OPENSMT_GIT_DESCRIPTION "\n";
+    std::cerr << version_string;
+    exit( 1 );
+}
+
+void parseCMDLine( int argc, char * argv[ ] )
+{
+    for ( int i = 1 ; i < argc ; i ++ )
+    {
+        const char * buf = argv[ i ];
+        if ( strcmp( buf, "--version" ) == 0
+            || strcmp( buf, "-v" ) == 0 )
```

Comment:
`-v` is used for verbosity, it should not be used for version.

Reply:
Fixed